src/errors.ts

```diff
@@ -171,7 +172,7 @@ export function getError(code?: FailureReason, message?: string) {
   return new UnspecifiedError(message);
 }
 
-export function getTransactionError(transaction: Payment | Transfer) {
+export function getTransactionError(transaction: Payment | Transfer | Remit) {
   const error: MtnMoMoError = getError(transaction.reason as FailureReason);
```

Comment:
No new remit specific errors to add to the `getError` function?

Reply:
to my knowledge, the existing errors cover the `remittances` edge cases. please advise if it is not the case.

Reply:
Did you check in the docs?